napari/_vispy/_tests/test_utils.py

```diff
@@ -60,11 +63,31 @@ def test_get_view_direction_in_scene_coordinates_2d(make_napari_viewer):
     viewer.dims.ndisplay = 2
 
     # get the viewbox
-    view_box = viewer.window._qt_viewer.view
+    view_box = viewer.window._qt_viewer.canvas.view
 
     # get the view direction
     view_dir = get_view_direction_in_scene_coordinates(
         view_box, viewer.dims.ndim, viewer.dims.displayed
     )
 
     assert view_dir is None
+
+
+def test_set_cursor(make_napari_viewer):
+    viewer = make_napari_viewer()
+    # cursor.value
+    # for cursor in CursorStyle
+    # if cursor != "invalid":
```

Comment:
Thanks for the simpler and faster test! I think you just need to remove these commented out lines.

Reply:
hehe oops